siliconcompiler/report/streamlit_viewer.py

```diff
@@ -309,7 +302,7 @@ def show_metrics_for_file(chip, step, index):
             streamlit.warning("This file does not include any metrics.")
 
 
-def show_manifest(manifest, max_num_of_keys_to_show=20):
+def show_manifest(chip, manifest, max_num_of_keys_to_show=20):
```

Comment:
Wouldn't it make more sense to have filtering enabled/disabled in your `make_manifest` function?
If this is way easier feel free to leave it though.

Reply:
I guess it depends on whether you want the user to allow filtering the "raw" manifest as well or just the "clean" manifest.
@gadfort Would filtering on the "raw" manifest provide any value to the user or is it too overwhelming?

Reply:
Nevermind filtering comes later so this works.